### DISCUSSION TITLE
[HELP] Ho fuso il citofono per renderlo Smart!

### DISCUSSION THREAD

**Discussion by odhinn33** [indomus/forum#2900] [GENERALE - Discussioni generiche]
Ciao a tutti,

ho un'appartamento in condominio e vorrei poter aprire da remoto 2 ingressi tramite citofono. Uso già Nuki Opener in altra casa, ma questo modello non è compatibile, pertanto devo trovare una soluzione alternativa. Tutto parte dal fatto che sono DUE i pulsanti di apertura sul citofono attuale. Il citofono attuale è il Bticino 344032 con sistema a 2 fili, ma dopo qualche esperimento goffo* l'ho fritto, quindi devo sostituirlo. Le soluzioni a cui ho pensato sono:

1. Comprare su ebay a 60€ stesso modello di citofono da fondo di magazzino (fuori produzione), acquistare il Relè MHCOZY 2ch 85-250V  e saldare i fili dei due canali ai contatti F-H e B-D (vedi foto) che sono direttamente dietro ai due pulsanti (cerchio blu e rosso). Ma ci sarebbero anche E-G ed A-C, quindi anche qui chiedo a voi. Inoltre spero di saper configurare correttamente il citofono sostitutivo per il pulsante secondario, ecc. Dovrò anche portare l'alimentazione fino lì, ma c'è un interruttore vicino quindi dovrei farcela con una piccola traccia.

2. Fare la stessa cosa ma su citofono più moderno e più compatibile per questa operazione Sprint L2 Bticino, sperando di trovare i collegamenti dietro i pulsanti facilmente senza friggerlo.

3. Acquistare un citofono compatibile con Ring Intercom di Amazon. Questo sistema aprirebbe solo UNA delle due porte e dovrei trovare il modo di far aprire solo il pulsante secondario, visto che il primo cancellino si può aprire anche con codice esterno. Eviterei questa soluzione perché comporterebbe andare ancora "alla cieca" con possibilità di friggere un secondo citofono oltre a risolvere solo parzialmente l'esigenza. 😅

4. Mi sono rivolto a due o tre professionisti ma tutti mi propongono di montare un sistema che lo permette nativamente tramite videocitofono costoso Videocitofono WiFi connesso BTICINO 2 fili anche se il condominio non ha videocamera. Ha senso? L'operazione mi costerebbe 550€. Vorrei poter trovare qualche elettricista un po' esperto di domotica ma non ne trovo. Per esempio i sistemi Sonoff so che sono molto utilizzati ma ce ne sono 100 e non saprei come muovermi. Ci potrei anche provare con qualche suggerimento. Inoltre se riuscissi (senza far danni) a farlo da solo il risparmio sarebbe notevole.

Tutte queste sono considerazioni di un NON professionista, lo ammetto, ed è proprio qui che chiedo suggerimenti o insulti se quello che voglio fare lo trovate sbagliato. Grazie a chiunque voglia dirmi la sua.

![IMG_20240131_134433](https://github.com/indomus/forum/assets/62665887/edc479bf-ee90-4a3b-8883-ad275f16eee8)
![IMG_20240131_134215](https://github.com/indomus/forum/assets/62665887/f7b9e93b-bbc2-4daa-90cf-b1578278e6c5)
![IMG_20240131_134442](https://github.com/indomus/forum/assets/62665887/b08b10c3-deca-4c49-ac32-a2f92c630912)

* l'ho fritto perché ho provato a collegare con tutte le varianti, le coppie di tutta la fila viola sperando di trovare quella corretta per la porta1 e la porta2. Una l'avevo anche trovata, ma cercando la seconda c'è stata una scintilla sospetta. Da quel momento il citofono ha smesso di funzionare. Solo dopo ho realizzato che potevo accontentarmi delle lettere rosse e blu. Ho provato anche a sostituire i fusibili verdi con il "3" e lo "0" ma nulla, morto. Sì, lo so, ho fatto una ca**ata. L'elettricista dice che ho rischiato di far saltare l'impianto del condominio 😨

**Comment by indomus** [comment]
NO LINK, grazie.

**Comment by odhinn33** [comment]
Ho rimosso i link. Qualche aiutino?

**Reply by alexnight80** [reply]
Ciao odhinn33, sei riuscito a risolvere il problema ? Quale soluzione hai scelto ? Io sono nella tua stessa condizione e non riesco ad uscirne fuori.
Cerco disperatamente aiuto !
Ho già acquistato un sonoff che consigliavano in un tutorial per modificare un citofono urmet classico, ma con questo Bticino non so proprio come collegarmi. Il mio pulsante apri porta è collegato ad un cancello elettrico le non vorrei bruciare tutto nei vari tentativi.
Ringrazio te e chiunque voglia aiutarmi.
Saluti

**Reply by odhinn33** [reply]
Ciao, leggo solo ora il tuo messaggio scusa! Io ho scelto la 4° soluzione trovando su subito un elettricista che aveva degli avanzi di magazzino ad ottimo prezzo. Poi me lo sono fatto mettere da un tecnico. Non volevo rischiare di fare altri danni. Ciao
